config/base/task-runner-rbac.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright The Conforma Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: conforma-vsa-generator
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This seems to duplicate the `ClusterRole`. I think it can be deleted.

Reply:
Agreed. Removed in a follow-up.